build/DotNetSdkVersions.txt

```diff
@@ -1,5 +1,5 @@
 # Each line represents arguments for the .NET SDK installer script (https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
--Channel 8.0 -Version 8.0.100-preview.6.23330.14
+-Channel 8.0 -Quality daily
```

Comment:
I'm not a fan that we have again "combined" SDK to test with, and SDK to test against. :( 

It leads to problems like this every single time.